paddlets/analysis/analysis_report.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         self._columns = columns
         self._analyzers = self._get_analyzers(names, params)
 
-    def export_docx_report(self, path: str = ".", file_name: str = "analysis_report.docx") -> None:
+    def export_docx_report(self, path: Optional[str] = ".", file_name: Optional[str] = "analysis_report.docx") -> None:
```

Comment:
should NOT be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional